.rubocop.yml

```diff
@@ -0,0 +1,34 @@
+require: rubocop-rails
```

Comment:
As a note, next time just copy these rules into a rubocop file to ignore the freeze ! error